privaterelay/glean/server_events.py

```diff
@@ -0,0 +1,133 @@
+"""
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+AUTOGENERATED BY glean_parser v11.0.1 for javascript_server.
+Converted to Python manually.
+TODO MPP-3642: Add python_server format to glean_parser.
```

Comment:
question (non-blocking): should we add a python_server format to glean_parser up-front? I.e., should the glean pings block and wait on a python server formatter? I'd hate to merge all this good code if it's going to have to be manually re-converted all over again?

Reply:
Short answer: no, we should not wait on `glean_parser` to add a `python_server` format. Let me know if you want a longer answer.